cypress/page/forms.page.ts

```diff
@@ -0,0 +1,70 @@
+class PersonalFormPage {
+    private firstName: string;
+    private lastName: string;
+    private email: string;
+    private gender: string;
+    private birthDate: string;
+    private mobile: string;
+    private hobbies: string;
+    private currentAddress: string;
+    private state: string;
+    private city: string;
+    private submit: string;
+    private formURL: string;
+    private modal: string;
+  
+    constructor() {
+      this.firstName = "#firstName";
+      this.lastName = "#lastName";
+      this.email = "#userEmail";
+      this.gender = "#genterWrapper";
+      this.mobile = "#userNumber";
+      this.birthDate = "#dateOfBirthInput";
+      this.hobbies = "#hobbiesWrapper";
+      this.currentAddress = "#currentAddress";
+      this.state = "#react-select-3-input";
+      this.city = "#react-select-4-input";
+      this.submit = "#submit";
+      this.formURL = "https://demoqa.com/automation-practice-form";
+      this.modal = "#example-modal-sizes-title-lg";
+    }
+  
+    public visitUrlPage(): void {
+      cy.visit(this.formURL);
+    }
+  
+    public fillForm(Info: PersonalInfo): void {
```

Comment:
parameters on methods should be in camelCase, so rename `Info` as `info` or `personalInfo`